docs/docs/configuration/genai.md

```diff
@@ -112,6 +112,7 @@ genai:
   provider: openai
   api_key: "{FRIGATE_OPENAI_API_KEY}"
   model: gpt-4o
+  base_url: https://api.openai.com/v1
```

Comment:
Including the `base_url` here implies it is required. It would be better to remove this and add a line below saying that `base_url` can be provided as an optional parameter.